crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -102,6 +102,12 @@ pub struct BloomSettings {
     /// configured in a non-energy-conserving way,
     /// otherwise set to [`BloomCompositeMode::EnergyConserving`].
     pub composite_mode: BloomCompositeMode,
+
+    // Maximum size of each dimension for the largest mipchain texture used in downscaling/upscaling.
+    pub max_mip_dimension: u32,
+
+    // UV offset for bloom shader. Ideally close to 2. / max_mip_dimension.
```

Comment:
2. -> 2.0 or 2

Also, these fields should have comments saying you should only tweak them if you're seeing visual artifacts.